Library/Homebrew/requirements/macos_requirement.rb

```diff
@@ -0,0 +1,29 @@
+require "requirement"
+
+class MacOSRequirement < Requirement
+  fatal true
+
+  def initialize(tags = [])
+    @version = MacOS::Version.from_symbol(tags.first) unless tags.empty?
+    super
+  end
+
+  def minimum_version?
```

Comment:
`minimum_version_provided` or `passed` or `specified` something would be nicer to make it clear that this doesn't mean the minimum version is met.